xml-csharp/SelectingXmlNodesWithXpath/SelectingXmlNodesWithXpathTests/SelectingXmlNodesWithXpathTest.cs

```diff
@@ -0,0 +1,123 @@
+using System.Diagnostics;
+using System.Xml;
+using System.Xml.Linq;
+using Xunit;

```

Comment:
This should have shown as a warning in the IDE, we have this in the global using so it is not needed here. We could also move the others to global usings